src/Microsoft.Diagnostics.Tools.RuntimeClient/Eventing/EventPipeClient.cs

```diff
@@ -45,7 +45,7 @@ public static IEnumerable<int> ListAvailablePorts()
         /// <param name="configuration">buffer size and provider configuration</param>
         /// <param name="sessionId">session id</param>
         /// <returns>Stream</returns>
-        public static Stream CollectTracing(int processId, SessionConfigurationV1 configuration, out ulong sessionId)
+        public static Stream CollectTracing(int processId, SessionConfiguration configuration, out ulong sessionId)
```

Comment:
Your commit messages mentioned preserving back-compat so just wanted to mention we aren't (yet) under any obligation to preserve back-compat on these API typenames, other than inconvenience it creates to refactor our own tool code that uses them.